nl.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+
+"EDIT_STATION_WINDOW_NAME_PLACEHOLDER" = "Naam";
+"EDIT_STATION_WINDOW_URL_PLACEHOLDER" = "http://pad/naar/stream";
+"EDIT_STATION_WINDOW_IMAGE_ACCESSIBILITY_LABEL" = "Kies afbeelding";
+
+"ADD_COLLECTION_WINDOW_TITLE" = "Voeg nieuwe verzameling toe";
+"EDIT_COLLECTION_WINDOW_TITLE" = "Wijzig verzameling";
+
+"EDIT_COLLECTION_WINDOW_NAME_PLACEHOLDER" = "Mijn verzameling";
+
+"SIDEBAR_CONTEXT_EDIT_COLLECTION" = "Wijzig";
+"SIDEBAR_CONTEXT_DELETE_COLLECTION" = "Verwijder verzameling";
+"SIDEBAR_CONTEXT_SHARE_COLLECTION" = "Deel";
+
+"STATION_CONTEXT_EDIT_COLLECTION" = "Wijzig";
+"STATION_CONTEXT_DELETE_COLLECTION" = "Verwijder zender";
+"STATION_CONTEXT_MOVE" = "Verplaats naar Verzameling";
+
+"CONTEXT_ALERT_DELETE_COLLECTION_TITLE" = "Verwijder \"%@\"";
+"CONTEXT_ALERT_DELETE_COLLECTION_BODY" = "Bent u zeker dat u deze collectie en alle zenders erin wenst te verwijderen?";
+
+"CONTEXT_ALERT_DELETE_STATION_TITLE" = "Verwijder \"%@\"";
+"CONTEXT_ALERT_DELETE_STATION_BODY" = "Bent u zeker dat u deze zender wenst te verwijderen?";
+
+"COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "Geen zenders";
+
+"WELCOME_COPY" = "Welkom bij Broadcasts! Als u wenst, kunnen we enkele voorkeurszenders van bij ons toevoegen waarmee u uw bibliotheek kan starten. Of, als u dit verkiest, kunt u handmatig uw eigen stations toevoegen of kies uit duizenden stations van de lijst.";
+
+"WELCOME_UPSELL" = "Welkom bij Broadcasts! In deze gratis versie kunnen we enkele voorkeurszenderes van bij ons toevoegen waarmee u uw bibliotheek kan starten. Of, als u liever upgradet naar de betaalde versie, kunt u uw eigen stations handmatig toevoegen of kiezen uit duizenden zenders van de lijst.";
+
+"WELCOME_ADD_PRESETS" = "Voeg voorkeuren toe";
+"WELCOME_NO_THANKS" = "Nee, bedankt";
+
+"GENERATED_COLLECTION_TITLE" = "Mijn verzameling";
+
+"UPSELL_COPY" = "Bedankt om Broadcasts te gebruiken!\n\nAls u dit wilt, kunt u de geavanceerde functies van Broadcasts ontgrendelen met een in-app-aankoop.\n\nOf, als u dit wilt, kunt u de vooraf ingestelde zenders blijven gebruiken en bewerken!";
+"UPSELL_BUY" = "Koop (%@)";
+"UPSELL_NOT_AVAILABLE" = "Niet beschikbaar";
+"UPSELL_RESTORE" = "Herstel aankopen";
+"UPSELL_NO_THANKS" = "Nee, bedankt";
+
+"TOOLBAR_NEW_CATEGORY_TOOLTIP" = "Nieuwe verzameling";
+"TOOLBAR_ADD_STATION_TOOLTIP" = "Voeg zender toe";
+"TOOLBAR_SHOW_SOURCES_TOOLTIP" = "Toon verzamelingen";
+"TOOLBAR_TOGGLE_SIDEBAR_TOOLTIP" = "Toon of verberg navigatiekolom";
+"TOOLBAR_TOGGLE_VIEW_TOOLTIP" = "Wissel tussen lijst- of rasterweergave";
+
+"IMAGE_WELL_REMOVE" = "Verwijder";
+"IMAGE_WELL_PASTE" = "Plak";
+"IMAGE_WELL_COPY" = "Kopieer";
+
+"CARPLAY_EMPTY_LIBRARY" = "Geen verzamelingen";
+"CARPLAY_ALL_STATIONS" = "Alle zenders";
+
+"MAC_ADD_STATION_BUTTON" = "Voeg zender toe";
+"MAC_EDIT_STATION_BUTTON" = "Wijzig zender";
+
+"MAC_ADD_COLLECTION_BUTTON" = "Voeg verzameling toe";
+"MAC_EDIT_COLLECTION_BUTTON" = "Wijzig verzameling";
+
+"STATION_DIRECTORY_WINDOW_TITLE" = "Zenderbrowser";
```

Comment:
This one is tricky. "Zenderbrowser" seems to be the logical translation.

But since we're talking about the directory an alternative translation is possible: the "iTunes Podcast Directory" is translated by Apple to "iTunes podcastoverzicht", which would make "zenderoverzicht" an alternative translation here.